.github/clear-street/README.md

```diff
@@ -0,0 +1,66 @@
+# Clear Street Proto with Buf
+
+## Goal
+
+At Clear Street, protobuf on Kafka is enabled with some special Kafka Headers or topic names.
+We've added [Buf](https://buf.build/) support to Kafka UI, as well as special deserialization logic.
+
+Use the [Protobuf on Kafka](https://www.notion.so/clearstreet/Protobuf-on-Kafka-ac912401cf504c75956d907b86a12f55) to reference the format.
```

Comment:
this is an internal document though and this repo is public. I don't think it is a good idea to link something that others can't access in a public README.md

Reply:
I think it's safe; it's behind a Notion login

Reply:
Agree for now. Eventually since this is open source/if we wat to merge upstream we'll want to clean it up

Reply:
yeah it is behind a Notion login, but I was against having something internal here. It is just a format, we can mention that here. Copy paste it from the doc. Kind of cleans up in advance if we want in upstream or not.